qutebrowser/config/configfiles.py

```diff
@@ -566,12 +566,18 @@ def _handle_error(self, action: str, name: str) -> typing.Iterator[None]:
 
     def finalize(self) -> None:
         """Do work which needs to be done after reading config.py."""
+        if config.instance.warn_autoconfig:
```

Comment:
```suggestion
        if self._config.warn_autoconfig:
```

for consistency with how the config is accessed elsewhere in this class.